usaspending_api/recipient/v2/views/states.py

```diff
@@ -112,15 +111,29 @@ def get_all_states(year=None, award_type_codes=None, subawards=False):
             .values("pop_state_code", "total", "distinct_awards", "outlay_total")
         )
 
-        results = [
-            {
-                "pop_state_code": row["pop_state_code"],
-                "total": row["total"],
-                "count": len(set(row["distinct_awards"].split(","))),
-                "total_outlays": row["outlay_total"],
-            }
-            for row in list(queryset)
-        ]
+        if len(queryset) > 0:
```

Comment:
I think this should be expanded to not just if there are no results, but even if there is only a single result. This list of states is the main entry point into the State Profile page which allows users to view more than just the latest FY.

Reply:
Updated this to check for more than 1 result.